mfjcs-rest/src/main/java/org/mfjcs/core/ItemStoreImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.mfjcs.core;
+
+import static org.joda.time.format.ISODateTimeFormat.basicDate;
+import static org.joda.time.format.ISODateTimeFormat.basicDateTime;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.mfjcs.api.Item;
+import org.mfjcs.api.ItemStore;
+import org.mfjcs.api.ItemStoreException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.dropwizard.jackson.Jackson;
+
+public class ItemStoreImpl implements ItemStore {
```

Comment:
Yes. At this point a core architecture brainstorm + research session seems
crucial

On Wed, 24 Aug 2016 13:14 Marc Portier, <notifications@github.com> wrote:

> In mfjcs-rest/src/main/java/org/mfjcs/core/ItemStoreImpl.java
> <https://github.com/marc-portier/mfjcs/pull/3#discussion_r76037872>:
>
> > +
> > +import java.io.BufferedWriter;
> > +import java.io.IOException;
> > +import java.nio.file.Files;
> > +import java.nio.file.Path;
> > +import java.nio.file.Paths;
> > +
> > +import org.mfjcs.api.Item;
> > +import org.mfjcs.api.ItemStore;
> > +import org.mfjcs.api.ItemStoreException;
> > +
> > +import com.fasterxml.jackson.
>
> databind.ObjectMapper;
> > +
> > +import io.dropwizard.jackson.Jackson;
> > +
> > +public class ItemStoreImpl implements ItemStore {
>
> other thought in the same area for the store you already model out
> interface/impl
> not so for the indexer (--> solr focus)
>
> not urgent, just saying - or to discuss a core arch decision?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/marc-portier/mfjcs/pull/3/files/a9dbe4de4a65060068a13809d530a398c07cf6e0#r76037872>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJVvUWoQCwHh-MCb45xV_BoAIuazZgjks5qjCeQgaJpZM4JrXRl>
> .
>